crafter-search-solr-server/src/main/resources/crafter/search/server-config.properties

```diff
@@ -94,4 +94,9 @@ crafter.search.parse.limit=5000
 
 # Search management authorization token
 # Please update this per installation and provide this token to the status monitors.
-crafter.search.management.authorizationToken=defaultManagementToken
\ No newline at end of file
+crafter.search.management.authorizationToken=defaultManagementToken
+
+# The name of the parameter for the access token
+crafter.search.api.access.token.parameter=token
+# The value of the access token
+crafter.search.api.access.token.value=defaultAccessToken
```

Comment:
shouldn't this be empty by default so we don't break current installations?

Reply:
I don't think so, the point of the fix is to be protected out of the box. Right @sumerjabri ?

Reply:
Correct.